packages/khalti_flutter/lib/src/page/bank_payment_page.dart

```diff
@@ -156,22 +161,23 @@ class _BankTile extends StatelessWidget {
               child: KhaltiColor(
                 isDark: isDark,
                 child: _BankBottomSheet(
-                  logo: bank.logo,
-                  name: bank.name,
+                  logo: widget.bank.logo,
+                  name: widget.bank.name,
                   amount: config.amount,
                   onTap: (mobile) async {
                     final url = Khalti.service.buildBankUrl(
-                      bankId: bank.idx,
+                      bankId: widget.bank.idx,
                       mobile: mobile,
                       amount: config.amount,
                       productIdentity: config.productIdentity,
                       productName: config.productName,
-                      paymentType: paymentType,
+                      paymentType: widget.paymentType,
                       productUrl: config.productUrl,
                       additionalData: config.additionalData,
                       returnUrl: config.returnUrl,
                     );
                     await urlLauncher.launch(url);
+                    if (!mounted) return;
                     Navigator.pop(context);
```

Comment:
Let's do like this instead.

```dart
Builder(
  builder: (context) {
    final navigator = Navigator.of(context);
    
    Builder(
      builder: (ctx) async {
        navigator.pop();
     }
   )
  }
)

```

Reply:
Should I do like this only in this file or in every file where I had made changes?